core/src/toga/fonts.py

```diff
@@ -55,51 +66,45 @@ def __str__(self) -> str:
         return f"{self.family} {size}{weight}{variant}{style}"
 
     @staticmethod
-    def register(family, path, weight=NORMAL, style=NORMAL, variant=NORMAL):
-        """Registers a file-based font with its family name, style, variant
-        and weight. When invalid values for style, variant or weight are passed,
-        ``NORMAL`` will be used.
+    def register(family, path, *, weight=NORMAL, style=NORMAL, variant=NORMAL):
+        """Registers a file-based font.
+
+        **Note:** This is not currently supported on macOS or iOS.
+
+        When invalid values for style, variant or weight are passed, ``NORMAL`` will be
+        used.
 
-        When a font file includes multiple font weight/style/etc, each variant must be
-        registered separately::
+        When a font includes multiple weights, styles or variants, each one must be
+        registered separately, even if they're stored in the same file::
+
+            from toga.style.pack import BOLD
 
             # Register a simple regular font
-            Font.register("Font Awesome 5 Free Solid", "resources/Font Awesome 5 Free-Solid-900.otf")
+            Font.register(
+                "Font Awesome 5 Free Solid", "Font Awesome 5 Free-Solid-900.otf"
```

Comment:
I included the `resources/` path to make it clear that it was a directory location.

Reply:
OK, I've restored it.